### DISCUSSION TITLE
小模型如何在CPU环境下运行

### DISCUSSION THREAD

**Discussion by SunYusheng** [OpenBMB/CPM-Live#344] [Q&A 问答]
感谢提供了CPM-Ant压缩版模型,但我只知道加载模型要依赖cpm_live.models,但安装时报错OSError: CUDA_HOME environment variable is not set. Please set it to your CUDA install root.
但用小模型的场景多半也没显卡, 那怎么在只有cpu的环境下使用呢? 能给出示例吗
我的环境是torch1.10.0+cpu

**Comment by zh-zheng** [comment]
这里的[`text_generation.py`](https://github.com/OpenBMB/CPM-Live/blob/cpm-ant/cpm-live/text_generation.py)可以使用CPU进行推理。
